packages/api/portfolioDrafts/application/createApplicationStep.ts

```diff
@@ -95,4 +96,5 @@ export const handler = middy(baseHandler)
   .use(jsonBodyParser())
   .use(validator({ inputSchema: wrappedApplicationStepSchema }))
   .use(JSONErrorHandlerMiddleware())
-  .use(cors(CORS_CONFIGURATION));
+  .use(cors(CORS_CONFIGURATION))
+  .use(IpCheckerMiddleware());
```

Comment:
We probably want this to always be the _frist_ middleware that gets executed so that the logging happens almost immediately at the start of the request. Thoughts @juliusfitzhugh-ccpo and @zachclark-ccpo?

Reply:
I agree, it should be the first middleware called.

Reply:
Yeah, if we want to log before anything happens it would be best to be the first middleware.